ppr-ui/src/layouts/LayoutPublic.vue

```diff
@@ -10,21 +10,18 @@
 
 </template>
 
-<script>
-    import SbcHeader from 'sbc-common-components/src/components/SbcHeader.vue'
-    import SbcFooter from 'sbc-common-components/src/components/SbcFooter.vue'
+<script lang="ts">
+  import {Component, Vue} from "vue-property-decorator";
+  import SbcHeader from 'sbc-common-components/src/components/SbcHeader.vue'
+  import SbcFooter from 'sbc-common-components/src/components/SbcFooter.vue'
 
-    export default {
-        name: 'LayoutPublic',
-        components: {
-            SbcHeader,
-            SbcFooter,
-        },
-        data: function () {
-            return {
-            }
-        },
-        inject: ['authApiUrl', 'originUrl'],
-    }
+  @Component({
```

Comment:
It's not all or nothing for Vue Composition.  In this case, I didn't see any benefit to a setup method, as is would actually be less efficient just for "inject" calls.

However, retrospectively, I should still use `createComponent` rather than the decorator, for both consistency and matching the direction of Vue moving forward.